web_client/views/ControlWidget.js

```diff
@@ -42,7 +42,7 @@ var ControlWidget = View.extend({
         if (options && options.norender) {
             return this;
         }
-        this.$el.html(this.template()(this.model.attributes));
+        this.$el.html(this.template()(this.model.attributes));  // eslint-disable-line backbone/no-view-model-attributes
```

Comment:
Backbone says the correct way is to [`_.clone(model.attributes)`](http://backbonejs.org/#Model-attributes).  The `toJSON` function doesn't actually list getting a copy as a purpose of the function, though it will do it (so, while `toJSON` works, it isn't the "correct" way -- just using the attributes works, but it isn't the correct way, either).

Our eslint rules complain on `_.clone(model.attributes)`, so rather than introduce changes in this PR that are other than eslint related, I've left it as is with a rule disable.